nncf/parameters.py

```diff
@@ -31,6 +31,7 @@ class TargetDevice(Enum):
     CPU = 'CPU'
     GPU = 'GPU'
     VPU = 'VPU'
+    CPU_SPR = 'CPU_SPR'
```

Comment:
Note that without additional changes the `"target_device": "CPU_SPR"` won't be parseable via NNCF .json config file. Furthermore, consider not failing with "CPU_SPR" on other backends, and instead mapping this value to regular "CPU" with a warning-level log message spawned.

Reply:
Thanks for the comment. Please, take a look.